src/backend/renderer.js

```diff
@@ -828,6 +818,18 @@ export function attach(
     }
   }
 
+  function unmountFiberChildrenRecursively(fiber: Fiber) {
+    if (__DEBUG__) {
+      debug('unmountFiberChildrenRecursively()', fiber);
+    }
+    let child = fiber.child;
+    while (child !== null) {
+      recordUnmount(child);
```

Comment:
Shouldn't we be checking if it's a fiber that we actually told the store about to begin with before recording an unmount?

We do this before recording a mount:
https://github.com/bvaughn/react-devtools-experimental/blob/456e937b83b8c05b0f7cdd6ea0e464018f5d9f19/src/backend/renderer.js#L793-L796

**Edit** Ah, never mind. The `recordUnmount` function does this for us:
https://github.com/bvaughn/react-devtools-experimental/blob/456e937b83b8c05b0f7cdd6ea0e464018f5d9f19/src/backend/renderer.js#L754

I get why these are inconsistent currently, but maybe we should them more consistent at some point to avoid confusion.